sparql/obsolete_replaced-by_terms_by_ontology.rq

```diff
@@ -0,0 +1,53 @@
+#+ summary: Find replaced terms by ontology
+#+ description: Choose an ontology and get back list of all obsolete terms with their terms replaced by it
```

Comment:
Change to:
`#+ description: Choose an ontology and get back a list of all obsolete terms with the terms that replace them`